src/js/letters/containers/AddressSection.jsx

```diff
@@ -142,10 +142,12 @@ export class AddressSection extends React.Component {
       // Reset all the error messages in case they go to edit again; should be pointless
       errorMessages: {}
     });
+    window.dataLayer.push({ event: 'letter-update-address-submit' });
```

Comment:
For consistency, you could probably throw this into the `saveAddressPending` action creator like you did with `success` and `failure`. Is there a reason why you opted to put this one in the component instead?

Reply:
Yeah fair enough, pushed a commit to move it.

I added that one first and didn't know i'd be touching actions at the time. But that makes sense.